src/applications/disability-benefits/526EZ/components/FormStartControls.jsx

```diff
@@ -2,28 +2,37 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import LoadingIndicator from '@department-of-veterans-affairs/formation/LoadingIndicator';
+
 import SaveInProgressIntro from '../../../../platform/forms/save-in-progress/SaveInProgressIntro';
+import backendServices from '../../../../platform/user/profile/constants/backendServices';
 
 import { ITFErrorAlert, VerifiedAlert, UnauthenticatedAlert, UnverifiedAlert, ITFDescription } from '../helpers';
 
+
 export default function FormStartControls(props) {
-  const { user } = props;
+  const { user: { login: { currentlyLoggedIn }, profile: { verified, services } } } = props;
+  const { EVSS_CLAIMS } = backendServices;
 
   const somethingWentWrong = props.ITFStatus && props.ITFStatus === 'expired' || // This may not be possible
                              props.errors && props.errors.length;
+  const hasEVSSClaimsService = !!(services.find(service => service === EVSS_CLAIMS));
+  const gateAuthenticatedUser = !verified && currentlyLoggedIn;
+  const gateVerifiedUser = !hasEVSSClaimsService && verified;
+  const permitVerifiedUser = verified && hasEVSSClaimsService;
   return (
     <div>
-      {!user.login.currentlyLoggedIn && <div>
+      {!currentlyLoggedIn && <div>
```

Comment:
What do you think of pulling this out of the return statement. Maybe in another function? Seems like if we're not careful with the block of conditionals above this, we could display two separate alerts (or an alert and a button) which is probably not what we're wanting.

Reply:
Good idea, will set `content` above and return content instead.